ansible/roles/ansible-tvault-contego-extension/templates/s3_others_compatible.conf.j2

```diff
@@ -1,26 +1,21 @@
 [DEFAULT]
-vault_storage_type = s3
-vault_storage_nfs_export = TrilioVault
-vault_data_directory_old = {{TVAULT_DATA_DIR_OLD}}
-vault_data_directory = {{TVAULT_DATA_DIR}}
-log_file = /var/log/nova/tvault-contego.log
-debug = False
-verbose = True
-max_uploads_pending = 3
-max_commit_pending = 3
 vault_s3_auth_version = DEFAULT
-vault_s3_access_key_id = {{VAULT_S3_ACCESS_KEY}}
-vault_s3_secret_access_key = {{VAULT_S3_SECRET_ACCESS_KEY}}
-vault_s3_region_name = us-east-1
-vault_s3_bucket = {{VAULT_S3_BUCKET}}
 vault_s3_endpoint_url = {{VAULT_S3_ENDPOINT_URL}}
 vault_s3_signature_version = {{VAULT_S3_SIGNATURE_VERSION}}
-vault_s3_support_empty_dir = {{VAULT_S3_SUPPORT_EMPTY_DIR}}
-vault_s3_ssl = {{VAULT_S3_SECURE}}
 
-[contego_sys_admin]
-helper_command = sudo {{privsep_helper_file}}
+vault_s3_ssl_cert = {{VAULT_S3_SSL_CERT}}
+vault_s3_ssl = {{VAULT_S3_SECURE}}
+vault_s3_access_key_id = {{VAULT_S3_ACCESS_KEY}}
+vault_s3_secret_access_key = {{VAULT_S3_SECRET_ACCESS_KEY}}
+vault_s3_region_name = {{VAULT_S3_REGION_NAME}}
+vault_s3_bucket = {{VAULT_S3_BUCKET}}
+vault_storage_type = s3
 
-[conductor]
-use_local = True
+vault_data_directory_old = {{TVAULT_DATA_DIR_OLD}}
+vault_data_directory = {{TVAULT_DATA_DIR}}
+log_file = /var/log/tvault-object-store/tvault-object-store.log
+debug = False
+verbose = True
 
+[s3fuse_sys_admin]
+helper_command = sudo {{privsep_bin_path.stdout}}
```

Comment:
create only one conf template file, no need to have so many different conf templates.

Reply:
changes done